lib/Dialect/FIRRTL/Transforms/EmitOMIR.cpp

```diff
@@ -1187,6 +1219,10 @@ void EmitOMIRPass::emitTrackedTarget(DictionaryAttr node,
       }
       target.push_back('>');
       target.append(addSymbol(componentName));
+
+      // If the target refers to a field, append the field.
+      if (hasFieldID(tracker))
+        target.append(addFieldID(componentType, tracker.fieldID));
```

Comment:
The `hasFieldID` check could be avoided if `addFieldID` was built to work (not assert) on other types. (I think it already works right now without the assert!), i.e., it returns an empty string.

Reply:
Good point, I'll update it.